pimpmycause/profiles/migrations/0019_auto_20170215_1930.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
How come this PR has migrations? If in the end nothing was added or amended we can just delete these migration files.